src/tests/test_physicsReferences.lua

```diff
@@ -0,0 +1,26 @@
+local PhysicsReferences = require("world/physicsReferences")
+local WorldObjects = require("world/worldObjects")
+local test = require("vendor/lunatest")
+
+local t = {}
+
+local world = love.physics.newWorld()
+local fixture = love.physics.newFixture(
+  love.physics.newBody(world),
+  love.physics.newCircleShape(1)
+)
+
+function t.test_valid_fixtureType()
+  PhysicsReferences.setFixtureType(fixture, "general")
```

Comment:
You could loop through one of the tables that lists the types and check that they are all valid.

Reply:
Sure, I guess that would check that all the tables are filled in and match for each type. Currently such a test would fail on master for the camera type.

Reply:
Unfortunately, the lists of categories are local to physicsReferences.lua.

Reply:
I made a function in physicsReferences that returns all the categories and used it to implement this.